src/native-common/WebView.tsx

```diff
@@ -44,11 +47,24 @@ export class WebView extends React.Component<RX.Types.WebViewProps, RX.Types.Sta
                 onError={ this.props.onError }
                 onMessage={ this.props.onMessage ? this._onMessage : undefined }
                 testID={ this.props.testId }
-                mixedContentMode={this.props.mixedContentMode}
+                mixedContentMode={this._sandboxToMixedContantMode(this.props.sandbox)}
             />
         );
     }
 
+    private _sandboxToMixedContantMode = (sandbox?: Types.WebViewSandboxMode): MixedContentMode => {
```

Comment:
This doesn't need to be a lambda instance variable. It can just be a normal method declaration.